templates/content/node--localgov-blog-post--teaser.html.twig

```diff
@@ -0,0 +1,138 @@
+{#
```

Comment:
Is there any chance this template could use the `node--teaser.html.twig` template instead of being its own template?